torch/csrc/distributed/c10d/socket.cpp

```diff
@@ -458,6 +454,7 @@ class SocketListenOp {
   bool tryListen(const ::addrinfo& addr);
 
   template <typename... Args>
+  // NOLINTNEXTLINE(cppcoreguidelines-missing-std-forward)
   void recordError(fmt::string_view format, Args&&... args) {
     auto msg = fmt::vformat(format, fmt::make_format_args(args...));
```

Comment:
Wait, we want to forward these args to format right? We should do that instead of surpress the lint (so we can record uncopyable args here with recordError.

Reply:
I see, but in the official example of libfmt, fmt::make_format_args don't need std::forward.